internal/bind/params.go

```diff
@@ -23,8 +24,31 @@ var (
 	errMultipleQueryParameters = errors.New("only one query arg *table.QueryParameters allowed")
 )
 
+func asUUID(v interface{}) (*uuid.UUID, bool) {
+	if _, ok := v.(interface {
```

Comment:
Do you need check the interface really? Few lines below you check concrete types